packages/mcp-server-supabase/src/server.ts

```diff
@@ -346,7 +347,10 @@ export function createSupabaseMcpServer(options: SupabaseMcpServerOptions) {
             .describe('The service to fetch logs for'),
         }),
         execute: async ({ project_id, service }) => {
-          const now = Date.now();
+          const timestamp =
+            service === 'action'
+              ? new Date(Date.now() - 5 * 60 * 1000)
+              : undefined;
```

Comment:
Any comment we can add here to explain the reason for this (to our future selves 😄)?

Reply:
added comment :)